src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/ShiftService.kt

```diff
@@ -26,63 +24,85 @@ class ShiftService(private val prisonService: PrisonService,
         val start = fromParam.orElse(LocalDate.now(clock))
         val end = toParam.orElse(LocalDate.now(clock))
         val region = prisonService.getPrisonForUser()?.region
-        val details = csrClient.getDetailsForUser(start, end, region, authenticationFacade.currentUsername)
-        val detailsByDate = groupDetailsByDate(details)
+        val detailsByDate = groupDetailsByDate(start, end, region)
 
         return start.datesUntil(end.plusDays(1)).map { date ->
-            val detailsForDate = detailsByDate.getOrDefault(date, listOf())
+            val details = detailsByDate.getOrDefault(date, listOf())
+            val (shiftDetails, overtimeDetails) = details.partition { it.shiftType == DetailParentType.SHIFT }
+            val startAndFinishDetails = (getStartAndFinishDetails(date, shiftDetails) + getStartAndFinishDetails(date, overtimeDetails))
```

Comment:
Are you concating two collections together with a `+`? Can we use `.plus()` for consistency with the rest of the codebase?